src/einsteinpy/plotting/geodesics_static.py

```diff
@@ -45,10 +98,70 @@ def plot_attractor(self):
         if not self._attractor_present:
             self._draw_attractor()
 
-    def _draw_attractor(self, min_radius=10 * u.km):
-        radius = max(schwarzschild_radius(self.mass) * 10000, min_radius.to(u.km))
-        color = "#ffcc00"
-        self.ax.add_patch(mpl.patches.Circle((0, 0), radius.value, lw=0, color=color))
+    def __get_x_y(self, pos_vec, vel_vec, end_lambda, step_size):
+        swc = Schwarzschild.from_spherical(pos_vec, vel_vec, self.time, self.mass)
+
+        vals = swc.calculate_trajectory(
+            end_lambda=end_lambda, OdeMethodKwargs={"stepsize": step_size}
+        )[1]
+
+        time = np.array([coord[0] for coord in vals])
+        r = np.array([coord[1] for coord in vals])
+        theta = np.array([coord[2] for coord in vals])
+        phi = np.array([coord[3] for coord in vals])
+
+        x = r * np.cos(phi)
+        y = r * np.sin(phi)
+        self.xarr = x
+        self.yarr = y
+        return x, y
+
+    def mindist(self, x, y):
+        return math.sqrt(x * x + y * y)
+
+    def _draw_attractor(self):
+        """ 
+            Auto scaling according to plots
```

Comment:
Indentation. Plus wrong use of docstrings.

Have a look at numpy like docstrings